app/models/sorting.rb

```diff
@@ -28,10 +29,10 @@ def apply_to_scope(scope)
   private
 
   def column
-    SORT_COLUMNS.fetch(sort_by.to_sym)
+    SORT_COLUMNS.fetch(sort_by&.to_sym || DEFAULT_SORT_BY)
```

Comment:
I don't think this can happen. The `SortingParams` helper declares default values for `sort_by` and `sort_direction`.

Reply:
I agree thats what we expected, it was like the default was not being set. Hence this addition. @timpeat what are your thoughts?

Reply:
@zheileman if a sorting object is not passed to the params then the defaults do not get populated. I might try to see if we can force the sorting object to set a default empty hash and see if that then causes the default values to get set.

Reply:
If you instantiate without providing values then the attributes get their defaults. But if you purposely pass any value, including blank or nil, then the passed value prevails, even if it is `nil`

```ruby
> Sorting.new.sort_by
=> "submitted_at"

> Sorting.new(sort_by: nil).sort_by
=> nil
```

This is expected. But because `Sorting` is used with inputs from the API, and these are already whitelisted by Grape (`values`) and set a default (`default`) then is redundant.

Only in a test scenario this can be reproducible. Unless I'm missing something 😅

Reply:
Thanks for the explanation :)

Reply:
This is no blocker by the way! Just me rambling. I approved the PR already 😄

Reply:
Thank you - I think its worth thinking about this if we can it would be cleaner if we we're able to rely on the defaults being in one place rather than two.

Reply:
@zheileman I'm going to do this^^^ 🦆

Reply:
That is a weird Grape behaviour. It does not happen in the GET /applications endpoint, I think this is because of the wrapping of `use :sorting` in an `optional :sorting`?
Maybe there is another way to do this, but don't know top of my head.

Feel free to choose whatever works best for you 👍

Reply:
```ruby
          optional :sorting, type: JSON, desc: 'Sorting JSON.', default: Sorting.new.attributes do
            use :sorting
          end
```